src/rust-crypto/rust-crypto.ts

```diff
@@ -375,7 +375,7 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
         const encryptor = this.roomEncryptors[room.roomId];
 
         if (encryptor) {
-            encryptor.ensureEncryptionSession(this.globalBlacklistUnverifiedDevices);
+            encryptor.prepareForEncryption(this.globalBlacklistUnverifiedDevices);
```

Comment:
This looks unrelated to the logging change?

Reply:
No it's part of the same thing. `ensureEncryptionSession` has gained an extra param and become private. `prepareForEncryption` is a new wrapper for it.

Reply:
Great, thanks.